src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.ParsedSyntaxTree.cs

```diff
@@ -121,6 +115,7 @@ public override CSharpParseOptions Options
                 }
             }
 
+            [Obsolete]
```

Comment:
With a message please.

Reply:
The intention is to have these be errors?

---
In reply to: [460199016](https://github.com/dotnet/roslyn/pull/44331#discussion_r460199016) [](ancestors = 460199016)